client/app/scripts/superdesk-search/views/search-facets.html

```diff
@@ -13,7 +13,7 @@
 
         <div sd-search-within></div>
 
-        <div sd-toggle-box data-title="{{ 'Parameters' | translate }}" data-open="false">
+        <div id="search-parameters" sd-toggle-box data-title="{{ 'Parameters' | translate }}" data-open="false">
```

Comment:
We could use one-time binding here... or is `translate` a stateless filter? We can use it anyway, just to be safe...